readme.txt

```diff
@@ -125,6 +125,10 @@ Some hosting providers filter this out using mod_security. For one user, they ne
 
 == Upgrade Notice ==
 
+= 3.3.3 =
+
+Due to the possibility of someone setting the url in their user profile to the same as another account, you will no longer be able to save the exact same url into two accounts
```

Comment:
what if two already use the same domain?

Reply:
The system would discard the second one and give anyone access to the first. But, you think it should be flagged and addressed?

Reply:
Shouldn't it throw an error if two users use the same URL? Otherwise it still is a security issue!